src/TsTypes.ts

```diff
@@ -1,5 +1,9 @@
 import { camelCase, upperFirst } from 'lodash'
 
+const multiLineCommentStart =  '/** '
+const multiLineCommentIndent = '    '
```

Comment:
I just tried to create a new issue to ask for a feature request to only use jsdoc comments for `descriptions`, before I saw this...

I don't know how you tried it, but VSCode works wonderfully if you use `/** */` in interfaces. It would be _really_ useful, if we could move descriptions to `/** */` comments.
![image](https://cloud.githubusercontent.com/assets/1152805/18661216/e2d70eca-7f14-11e6-9fae-4bd60d6fc938.png)